.github/label-actions.yml

```
@@ -274,3 +274,25 @@
 
 
     Thanks, the Renovate team
+
+'auto:one-topic':
+  comment: >
+    Hi there,
+
+    Please stick to one topic/question per Discussion. Start a new discussion per topic.
+
+    This is to benefit future users who may search and find this discussion and not be confused because there's multiple questions and answers within. One question and answer per Discussion works best.
+
+    For example:
+     - Don't create a discussion with multiple questions requiring multiple answers
+     - If you got an answer to your first question, don't change topic like "Can I also ask you how I might..." which would then require a different answer. Mark the current discussion as answered and open a new one.
+
+'auto:no-mentions':
+  comment: >
+    Hi there,
+
+    Please do not unnecessarily `@` maintainers like `@rarkins` or `@viceice`. Doing so causes annoying notifications and make maintaining a repo of this size difficult.
+
+    For example, never `@` mention a maintainer when you are creating a discussion if your desire is to get attention. This is inconsiderate behavior, just like shouting out your coffee order in a Starbucks before it's your turn.
```

```suggestion
    For example, never `@` mention a maintainer when you are creating a discussion if your desire is to get attention. This is rude behavior, just like shouting out your coffee order in a Starbucks before it's your turn.
```

`Inconsiderate` is a difficult word, I think `rude` or `bad` is easier to understand.